roles/certificates/tasks/main.yml

```diff
@@ -47,15 +47,24 @@
     group: root
     mode: '0644'
 
+- name: 'Create CA key password file'
+  ansible.builtin.copy:
+    content: "{{ certificates_ca_password }}"
+    dest: "{{ certificates_ca_directory_keys }}/ca.pwd"
+    owner: root
+    group: root
+    mode: '0600'
+  no_log: true
+
 - name: 'Creating CA certificate and key'
   ansible.builtin.command: >
     openssl req -new
       -x509
-      -nodes
```

Comment:
@ehelms why was this here?

Reply:
I don't remember, this role is basically a lift of https://github.com/theforeman/forklift/blob/master/roles/custom_certificates/tasks/main.yml